src/components/DashboardLeft.jsx

```diff
@@ -0,0 +1,17 @@
+import LocationRow from "src/components/LocationRow";
+import { Grid } from '@mui/material';
+
+
+const Left = ({locations, selectedLocation}) => {

```

Comment:
see comments for `Dashboard` if this should be its separate component

Reply:
I think it's more readable this way, but we could move it if it results in  slightly lower performance?

Reply:
then feel free to revert the changes